docs/src/main/asciidoc/getting-started-reactive.adoc

```diff
@@ -46,22 +47,30 @@ The solutions are located in the `getting-started-reactive` and `getting-started
 Quarkus is reactive.
 If you look under the hood, you will find a reactive engine powering your Quarkus application.
 This engine is Eclipse Vert.x (https://vertx.io).
-Every IO interaction passes through the non-blocking and reactive Vert.x engine.
+Every network I/O passes through the non-blocking and reactive Vert.x engine.
 
-image:quarkus-reactive-stack.png[alt=Quarkus is based on a reactive engine, 50%]
+image:quarkus-reactive-stack.png[alt=Quarkus is based on a reactive engine, 70%]
 
 Let's take 2 examples to explain how it works.
 Imagine an incoming HTTP request.
-The (Vert.x) HTTP server receives the request and then routes it to the application.
-If the request targets a JAX-RS resource, the routing layer invokes the resource method in a worker thread and writes the response when the data is available.
+The (Vert.x) HTTP server, embedded in Quarkus, receives the request and then routes it to the application.
+If the request targets an _imperative_ method (traditional Jax-RS, code annotated with `@Blocking`...), the routing layer invokes the resource method in a _worker_ thread and writes the response when the data is available.
```

Comment:
It's minor but isn't it `JAX-RS`? Same comment for the rest of the document.

Reply:
Yes!